nncf/common/pruning/schedulers.py

```diff
@@ -128,13 +162,13 @@ class ExponentialWithBiasPruningScheduler(PruningScheduler):
     P - pruning rate for current epoch
     a, b, k - params
     """
-    def __init__(self, pruning_algo, config=None):
-        super().__init__(pruning_algo, config)
-        self.a, self.b, self.k = self._init_exp(self.pruning_steps, self.initial_pruning, self.pruning_target)
+    def __init__(self, controller, params: dict):
+        super().__init__(controller, params)
+        self.a, self.b, self.k = self._init_exp(self.num_pruning_epochs, self.initial_pruning, self.target_pruning)
 
-    def _calc_pruning_level(self):
-        curr_pruning = self.a * np.exp(-self.k * (self.current_epoch - self.num_init_steps - 1)) + self.b
-        max_pruning = self.pruning_target
+    def _calculate_pruning_level(self) -> float:
+        curr_pruning = self.a * np.exp(-self.k * (self.current_epoch - self.num_warmup_epochs - 1)) + self.b
+        max_pruning = self.target_pruning
         return max_pruning if curr_pruning >= max_pruning else curr_pruning
 
     @staticmethod
```

Comment:
Could you format docstring for  `def _init_exp(E_max, P_min, P_max, D=1 / 8)` method

Reply:
Done